### ISSUE TITLE
Suggestion / Discussion : "Propose Change", "Suggest merge" in addition to "Challenge"

### ISSUE THREAD

**Issue by Web3Slimchance** [graphprotocol/mission-control-curator#113]
We need more nuance to the challenge system. 

A typo in one of the links is significant, and should be corrected. It still feels a bit harsh to vote for removal. Instead we should be able to **"Propose A Change"**. If the project owner agrees to the proposal, the project would update. If the project owner is absent, this could be decided by vote.

Another issue is with duplicate entries. Sometimes it is easy to determine which one to remove. Other times, it is more challenging to determine which one to keep.
The oldest entry?
The most up to date entry?
The entry with the most relevant information?
The entry with accurate links?
The best would be to have a **"Suggest Merge"** option.

This also ties with my thread on the credibility system https://github.com/graphprotocol/mission-control-curator/issues/14. It feels wrong to remove someones "voting and credibility score" within the Everest system, just because they have made a typo etc. 

Please feel free to discuss.

**Comment by LeapGamer** [comment]
Agree on the need to be able to merge, but for now I'd say that the entry with the most accurate information should always win.

**Comment by IrinaLavrinova** [comment]
I also think that the moment of editing should be present. Anyone can make a mistake, it is a human factor.

**Comment by Web3Slimchance** [comment]
> Agree on the need to be able to merge, but for now I'd say that the entry with the most accurate information should always win.

It is really hard to judge which entry is the "most accurate";
Entry A : Wrong github link
Entry B : Wrong homepage link
Entry C : Inaccurate/outdated project description

Which is why we need a more nuanced edit/proposal system. Maybe we could propose changes field by field. etc

**Comment by GLCNI** [comment]
I created x2 Challenges recently, for a Spelling mistake and to add a category to the project as it was blank. I don't want these projects to be removed and if they are voted to be Kept then I lose the Deposit. I seen it as a sunk cost and the best way so far to highlight the issue, however this definitely discourages challenges for edits. 
I would suggest if the owner does not make the edits within 7 days after the "propose change" is passed, then the challenger can make the edits.

**Comment by yanivtal** [comment]
I agree this is a great suggestion. One of the things we're lacking right now is a good communication system to make this type of coordination feasible. If we could send a message to the owner then suggesting a change would be easier. Let's work on adding messaging and then we could add this type of a feature in. I'm all for.
